.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
```

Comment:
Shouldn't we also set `max_line_length` for all files?

Reply:
Something like 80 or 100??, I would prefer 100, but some teams prefer 100

Reply:
100 is fine